src/theme/widget.rs

```diff
@@ -47,8 +47,55 @@ pub fn label(text: impl Into<String>) -> impl Bundle {
     )
 }
 
-/// A simple button with text and an action defined as an [`Observer`].
+/// A large rounded button with text and an action defined as an [`Observer`].
 pub fn button<E, B, M, I>(text: impl Into<String>, action: I) -> impl Bundle
+where
+    E: Event,
+    B: Bundle,
+    I: IntoObserverSystem<E, B, M>,
+{
+    button_base(
+        text,
+        action,
+        (
+            Node {
+                width: Px(300.0),
+                height: Px(80.0),
+                align_items: AlignItems::Center,
+                justify_content: JustifyContent::Center,
+                ..default()
+            },
+            BorderRadius::MAX,
+        ),
+    )
+}
+
+/// A small square button with text and an action defined as an [`Observer`].
+pub fn button_small<E, B, M, I>(text: impl Into<String>, action: I) -> impl Bundle
+where
+    E: Event,
+    B: Bundle,
+    I: IntoObserverSystem<E, B, M> + Sync,
```

Comment:
```suggestion
    I: IntoObserverSystem<E, B, M>,
```
The `+ Sync` isn't required here I believe? Same goes for `fn button`.